cairo_zero/kakarot/accounts/library.cairo

```diff
@@ -374,6 +374,35 @@ namespace AccountContract {
         Account_code_hash.write(code_hash);
         return ();
     }
+
+    // @notice Execute a starknet call.
+    // @dev Used when executing a Cairo Precompile. Used to preserve the caller address.
+    // @param to The address to call.
+    // @param function_selector The function selector to call.
+    // @param calldata_len The length of the calldata array.
+    // @param calldata The calldata for the call.
+    // @return retdata_len The length of the return data array.
+    // @return retdata The return data from the call.
+    // @return success 1 if the call was successful, 0 otherwise.
+    func execute_starknet_call{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
```

Comment:
no tests ?

Reply:
All existing tests using Cairo Precompiles are testing this